glmmTMB/R/VarCorr.R

```diff
@@ -345,8 +345,7 @@ format_corr.vcmat_ar1 <- function(x, maxdim = Inf, digits=2, ...) {
     } else {
         cc <- format(round(x[2,1], digits), nsmall = digits)
     }
-    cov_struct <- if (inherits(x, "vcmat_hetar1")) "(hetar1)" else "(ar1)"
-    return(matrix(paste(cc, cov_struct)))
+    return(matrix(paste(cc, "(ar1)")))
```

Comment:
is the idea here that it will be obvious to the user whether a homogeneous or a heterogeneous ar1 structure is being used (because hetar1 will print out multiple SD values while ar1 will only print one?)

Reply:
yeah that was my idea (this is how it was currently printed) - but of course would be good to align with how you treat other homogeneous / heterogeneous covariance pairs for the printing